### PR TITLE
Updated Description

### PR DESCRIPTION
**Pull request by MagdaMatetovici**

### CODE REVIEW

File: ShinyWheater/DESCRIPTION [hunk 1]
```diff
@@ -2,12 +2,15 @@ Package: ShinyWheater
...
-    person("Magda", "Matetovici", , "magda.matetovici@yahoo.com", role = c("aut", "cre"))
-Description: This app will show the weather in a Shiny app in a friendly manner for children.
-It will show temperature, wind, chances of rain.
-Depending on the temperature, wind and rain it will show a picture of 
-what can be done outside which can be refreshed to give several ideas.
+    person(given = "Magda",
+    family = "Matetovici", 
+    email = "magda.matetovici@yahoo.com", 
+    role = c("aut", "cre"))
+Description: This app will show the weather in a Shiny app in a friendly manner for children. It will show temperature, wind, chances of rain. Depending on the temperature, wind and rain it will show a picture of what can be done outside which can be refreshed to give several ideas.
```
Comment (mfhstdt) [inline ShinyWheater/DESCRIPTION:9]: For the description you should insert an indention of 4 spaces (like you did with Authors@R above)

**Review by mfhstdt — COMMENTED** [review]
Sounds like a cool project and the R package structure is not missing anything.
I spotted a missing indention in the DESCRIPTION file but otherwise all looks well!

File: ShinyWheater/DESCRIPTION (hunk 1 above)
Comment (MarliesRoozendaal) [inline ShinyWheater/DESCRIPTION:9]: I like your idea!

File: ShinyWheater/R/Main_Shiny.R
Comment (MarliesRoozendaal) [inline ShinyWheater/R/Main_Shiny.R:1]: You can choose beween one file that contains both ui and the server or two separate files. That might be something to think about:)

**Review by MarliesRoozendaal — COMMENTED** [review]
Good idea! It's not something that I have seen before, so I'm excited to see your final work.

**Pull request merged**